pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -62,13 +64,15 @@ func NewConfigObserver(
 				SchedulerLister:       configInformer.Config().V1().Schedulers().Lister(),
 
 				ConfigmapLister:              kubeInformersForNamespaces.ConfigMapLister(),
+				SecretLister_:                kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().Secrets().Lister(),
```

Comment:
why underscore?

Reply:
ic, name conflict

Reply:
can this be private?

Reply:
field and method (`SecretLister()`) cannot be the same